.github/workflows/vpm_ci.yml

```diff
@@ -0,0 +1,60 @@
+name: VPM CI
+
+on:
+  push:
+    paths:
+      - '**/vpm_ci.yml'
+      - 'cmd/tools/vpm.yml'
```

Comment:
hm, there is no `cmd/tools/vpm.yml` file?

Reply:
Do you mean this instead?
```suggestion
      - 'cmd/tools/vpm/**'
```

Reply:
oops, thank you!